apps/youtube_qa/multi_youtube_video_qa/multi_youtube_video_qa.py

```diff
@@ -0,0 +1,324 @@
+# coding=utf-8
```

Comment:
This script is based on an old version of youtube_qa.py. The script is now updated with new logics as evadb has improved since then. Perhaps incorporate some changes from youtube_qa.py in the new master branch.

Reply:
the new logic of summarization / blog post generation may not be needed currently for this app. Have rebased the other changes. Please let me know if there is anything else to incorporate